Documentation/helm-operator.md

```diff
@@ -54,12 +41,6 @@ For Helm `v3.x`:
 helm install --namespace rook-ceph rook-ceph rook-release/rook-ceph
```

Comment:
Please remove the `For Helm `v3.x`:` on line 38 as well

Reply:
done